src/v/config/configuration.h

```diff
@@ -387,6 +387,7 @@ struct configuration final : public config_store {
       kafka_quota_balancer_node_period;
     property<double> kafka_quota_balancer_min_shard_throughput_ratio;
     bounded_property<int64_t> kafka_quota_balancer_min_shard_throughput_bps;
+    property<std::unordered_set<int16_t>> kafka_throughput_controlled_api_keys;
```

Comment:
i think this could be a `vector<string>`, right? the kafka layer makes available all of the names of the kafka requests. so you could map the names to the numeric keys (more efficient at runtime) in the kafka layer rather than forcing the user to figure out the right numbers.

Reply:
Irrespective of the underlying type, we should use something with a consistent order (like a vector), because any tool trying to reconcile configs via an API is liable to make mistakes if it sees a different order in its GET than it provided in a PUT.

Reply:
excellent. seems we have this problem of preserving order now with the existing use of property<unordered_map> (not sure if it is actually used)--maybe should be removed?